contracts/token/BridgeMinter.sol

```diff
@@ -135,4 +124,11 @@ contract BridgeMinter is Manager {
     function getController() public view returns (address) {
         return address(controller);
     }
+
+    /**
+     * @dev Returns IBridgeMinterToken interface
+     */
+    function livepeerToken() private view returns (IBridgeMinterToken) {
+        return IBridgeMinterToken(IController(controller).getContract(keccak256("LivepeerToken")));
```

Comment:
```suggestion
        return IBridgeMinterToken(controller.getContract(keccak256("LivepeerToken")));
```

`controller` should already be a `IController` type.

Reply:
fixed in 0e00833